python/tank/pipelineconfig.py

```diff
@@ -741,6 +741,10 @@ def execute_core_hook_internal(self, hook_name, parent, **kwargs):
             # of the core API.
             hooks_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", "..", "hooks"))
             hook_path = os.path.join(hooks_path, file_name)
+        else:
+            # some hooks are always custom. ignore those and log the rest.
+            if hook_name not in ["pick_environment"]:
```

Comment:
minor tweak suggestion - could be nice to turn this into a constant perhaps and whack it in the constants file?